internal/config/config.go

```diff
@@ -67,6 +68,18 @@ type Egress struct {
 	// file, can only be set through the environment variable.
 	SnykServiceAccountToken string `json:"-" env:"SNYK_SERVICE_ACCOUNT_TOKEN"`
 }
+
+type Route struct {
+	// OrganizationID is the snyk organization ID where data should be routed to.
+	OrganizationID string `json:"organizationID"`
+	// ClusterResources when set to true, it will route cluster level resources
+	ClusterResources bool `json:"clusterResources"`
```

Comment:
Nit: I think naming this `ClusterScopedResources` would be better as it is the [standard that kubernetes](https://kubernetes.io/docs/reference/using-api/api-concepts/#resource-uris) uses